roles/cockpit-ui/tasks/main.yml

```diff
@@ -22,43 +22,45 @@
   changed_when: false
 
 - name: Create passthrough route for docker-registry
-  command: >
-    {{ openshift.common.client_binary }} create route passthrough
-    --service docker-registry
-    --config={{ openshift_hosted_kubeconfig }}
-    -n default
-  register: create_docker_registry_route
-  changed_when: "'already exists' not in create_docker_registry_route.stderr"
-  failed_when: "'already exists' not in create_docker_registry_route.stderr and create_docker_registry_route.rc != 0"
+  oc_route:
+    name: passthrough
+    namespace: default
+    service_name: docker-registry
+    kubeconfig: "{{ openshift_hosted_kubeconfig }}"
+    state: present
+  run_once: true
   when: check_docker_registry_exists.rc == 0
```

Comment:
Looks like it would be safe to remove the check then as well. Will do in the next set of updates.